### ISSUE TITLE
Lawn area behind 73-83 Blackthorn Road appartment block

### ISSUE THREAD

**Issue by Thurgarton66** [theparksilford/Issues#68]
### Property Details

73-83 Blackthorn Road

### Type of problem?

Ground Maintenance

### Since when this issue started?

17/05/2024

### Issue Description?

Can this area be mown/grass cut please. The grass is now long. 
![IMG_4591](https://github.com/theparksilford/Issues/assets/105227854/c4072ea6-7e20-40dc-b0da-d3893ea2d0eb)

### Logged By

Leaseholder or FreeHolder

**Comment by theparksilford** [comment]
Yes, this behind the block and hidden from the new guy. I have just showed it to him so he'll start doing it from next week.
